doc/userguide/src/Appendices/ReproducibleBuilds.rst

```diff
@@ -0,0 +1,9 @@
+Reproducible Builds
+===================
+
+Robot Framework honors the `SOURCE_DATE_EPOCH
+<https://reproducible-builds.org/docs/source-date-epoch/>` environment
+variable so that it can be built in a reproducible fashion.  When the
+`SOURCE_DATE_EPOCH` environment variable is set, timestamps embedded
+in generated documentation and log messages are set to its fixed
+value.
```

Comment:
Would BUILD.rst be better place for this information than the User Guide?

Reply:
BUILD.rst seems to be specific to creating Robot Framework releases.  The `SOURCE_DATE_EPOCH` environment variable is most useful for the (automated) build tooling of 3rd parties packaging RobotFramework such as GNU Guix, NixOS or Debian that strive for reproducible builds.  So it doesn't really need to be user-facing, but it's also not at home in BUILD.rst.  I'm not sure of the value to document it, but if we do, an appendix seems the best place to me (it's there if someone looks for it, but it's in the background as most users probably do not care).

Reply:
Isn't `SOURCE_DATE_EPOCH` exactly about creating releases? I don't see problems enhancing BUILD.rst to cover also third party packaging.